Filtration/Views/ItemFilterBlockView.xaml

```diff
@@ -130,48 +131,48 @@
                             <WrapPanel Grid.Row="0" Grid.Column="2" VerticalAlignment="Center">
                                 <Image Height="40" Width="40" Margin="0,0,10,0" Stretch="Fill">
                                     <Image.Source>
-                                        <MultiBinding Converter="{StaticResource MinimapIconToCroppedBitmapConverter}">
-                                            <Binding Path="DisplayIconSize"/>
-                                            <Binding Path="DisplayIconColor"/>
-                                            <Binding Path="DisplayIconShape"/>
+                                        <MultiBinding Converter="{StaticResource MinimapIconToCroppedBitmapConverter}">
+                                            <Binding Path="DisplayIconSize"/>
+                                            <Binding Path="DisplayIconColor"/>
+                                            <Binding Path="DisplayIconShape"/>
                                         </MultiBinding>
                                     </Image.Source>
                                 </Image>
                                 <Button Command="{Binding PlaySoundCommand}"
-                                        Width="25"
-                                        Height="25"
+                                        Width="32"
+                                        Height="32"
                                         VerticalAlignment="Center"
                                         HorizontalAlignment="Center"
                                         Margin="0,0,3,0"
                                         Visibility="{Binding HasSound, Converter={StaticResource BooleanVisibilityConverter}}"
                                         Background="Transparent"
                                         BorderBrush="Transparent"
                                         ToolTip="Click to preview drop sound">
```

Comment:
Should we be making the text of these tooltips conditional based on whether the block has DisableDropSound on it? Something like "Click to preview drop sound (Drop sound disabled for this block)" ?